yarl/_url.py

```diff
@@ -413,6 +423,16 @@ def __setstate__(self, state):
             self._val, *unused = state
         self._cache = {}
 
+    def _cache_netloc(self) -> None:
+        """Cache the netloc parts of the URL."""
+        cache = self._cache
+        (
+            cache["raw_user"],
+            cache["raw_password"],
+            cache["raw_host"],
+            cache["explicit_port"],
```

Comment:
These lines are covered with coverage and codecov is only reporting mypy doesn't see them